astroid/brain/brain_attrs.py

```diff
@@ -0,0 +1,38 @@
+"""Astroid hook for the attrs library
+
+Without this hook pylint reports unsupported-assignment-operation
+for atrrs classes
+"""
+import astroid
+from astroid import MANAGER
+
+
+def register(linter):
+  # Needed for registering the plugin.
+  pass
+
+
+def _attr_assign_attributes(node):
+    for cdefbodynode in node.body:
+        if not isinstance(cdefbodynode, astroid.Assign):
+            continue
+        for target in cdefbodynode.targets:
+            node.locals[target.name] = [astroid.Attribute(target.name)]
+
+
+def attr_attributes_transform(node):
+    """If the given ClassNode has an attr decorator,
+    then rewrite class attributes as instance attributes
+    """
+    if not node.decorators:
+        return None
+    for decorator_node in node.decorators.nodes:
+        if not isinstance(decorator_node, astroid.Attribute):
+            continue
+        if decorator_node.as_string() in ("attr.s",
```

Comment:
Please use `.qname()` rather than `as_string()`

Reply:
qname isn't available for Attribute nodes. I could do decorator_node.inferred()[0].qname() however that returns `attr._make.attributes`; this fully qualified name refers to a module that could be changed since `_make` isn't part of the API, so I thought that using the names would be better.

What do you think?

Reply:
Sounds good, we can use `as_string`